common/src/main/scala/org/apache/celeborn/common/metrics/source/AbstractSource.scala

```diff
@@ -257,6 +257,20 @@ abstract class AbstractSource(conf: CelebornConf, role: String)
     }
   }
 
+  override def decCounter(metricsName: String, decV: Long = 1): Unit = {
```

Comment:
Shall we just call `incCounter(xxx, -1)` to achieve this functionality, so we don't need to add these methods.